packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -17,9 +17,10 @@
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
     <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
     <!-- Start Custom .NET Native properties -->
-    <DotNetNativeVersion>2.2.7-rel-27913-00</DotNetNativeVersion>
+    <DotNetNativeVersion>2.2.8-rel-28605-00</DotNetNativeVersion>
+    <DotNetNativeRuntimeVersion>2.2.7-rel-28605-00</DotNetNativeRuntimeVersion>
```

Comment:
>DotNetNativeRuntimeVersion [](start = 5, length = 26)

(nit) why is this different from DotNetNativeVersion?

Reply:
I do not know. I guess someone forgot to increment the version number when released this package.